index.html

```diff
@@ -55,6 +55,162 @@ <h1>I’m Mariane Glad to see you!</h1>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#" class="project-details">See Project</a>
+            </div>
+        </div>
+
+        <div class="about">
+            <header class="header">
+                <p class="about-title">About Myself</p>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <a href="#">LET’S CONNECT</a>
+                <ul class="social">
+                    <li>
+                        <a href="#">
+                            <img src="images/Github.svg" alt="Github icon">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="images/Linkedin.svg" alt="Linkedin icon">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="images/Angelist.svg" alt="Angelist icon">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="images/Twitter.svg" alt="Twitter icon">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="images/Medium.svg" alt="Medium icon">
+                        </a>
+                    </li>
+                </ul>
+                <a href="#" class="resume-button">Get my resume</a>
+            </header>
+
+            <div class="languages">
+                <div class="item">
+                    <div>Languages</div>
+                    <img class="down-arrow" src="images/Down-arrow.svg" alt="Down-arrow icon">
+                </div>
+                <div class="content">
+                    <div>
+                        <img src="images/JS.png" alt="JS icon">
+                        <div>Javascript</div>
+                    </div>
+                </div>
+                <div class="content">
+                    <div>
+                        <img src="images/HTML5.png" alt="JS icon">
+                        <div>HTML</div>
+                    </div>
+                </div>
+                <div class="content">
+                    <div>
+                        <img src="images/CSS3.png" alt="JS icon">
+                        <div>CSS</div>
+                    </div>
+                </div>
```

Comment:
- [ ] For this section, wrap the various technologies using `<ul>` and `<li>` tags instead of the `<div>`. It ensures that you have enhanced your semantics.